behat.yml.dist

```diff
@@ -131,6 +131,23 @@ default:
             filters:
                 tags: "@inventory"
 
+        import_export:
+            contexts:
+                - Sylius\Bundle\ImportExportBundle\Behat\ImportExportContext
+                - Behat\MinkExtension\Context\MinkContext
+                - Sylius\Bundle\CoreBundle\Behat\HookContext
+                - Sylius\Bundle\AddressingBundle\Behat\AddressingContext
+                - Sylius\Bundle\CoreBundle\Behat\CoreContext
+                - Sylius\Bundle\PaymentBundle\Behat\PaymentContext
+                - Sylius\Bundle\ProductBundle\Behat\ProductContext
+                - Sylius\Bundle\TaxationBundle\Behat\TaxationContext
+                - Sylius\Bundle\TaxonomyBundle\Behat\TaxonomyContext
+                - Sylius\Bundle\WebBundle\Behat\WebContext
+                - Sylius\Bundle\MoneyBundle\Behat\MoneyContext
+                - Sylius\Bundle\ChannelBundle\Behat\ChannelContext
+            filters:
+                tags: "@import_export && ~@javascript"
```

Comment:
You don't need that `~@javascript` condition, because running JavaScript tests is based on current profile.